src/SWP/Bundle/ContentBundle/Controller/RoutesController.php

```diff
@@ -237,7 +238,7 @@ private function handleRouteUpdate($route, $routeData)
             $route->setName($routeData['name']);
         }
 
-        if (isset($routeData['type']) && $routeData['type'] == Document\Route::TYPE_CONTENT) {
+        if (isset($routeData['type']) && $routeData['type'] == Phpcr\Route::TYPE_CONTENT) {
```

Comment:
but hardcoding this to `Phpcr` don't make whole idea of switching engines out of the table? As switching it would require code changes? We should move it to some base class shared between implementations or to interface.